ui/packages/tidb-dashboard-lib/src/apps/UserProfile/components/Form.SSO.tsx

```diff
@@ -257,6 +257,27 @@ export function SSOForm() {
                     style={DEFAULT_FORM_ITEM_STYLE}
                   />
                 </Form.Item>
+                <Form.Item
+                  name="client_secret"
+                  label={t('user_profile.sso.form.client_secret')}
+                  rules={[{ required: false }]}
+                >
+                  <Input
+                    disabled={!isWriteable}
+                    style={DEFAULT_FORM_ITEM_STYLE}
+                  />
```

Comment:
This should be showed as password, not plain text. I will help refine it later.

Reply:
> This should be showed as password, not plain text. I will help refine it later.

Yes, you are correct. Thanks for pointing it out.